lib/cursor-position-view.coffee

```diff
@@ -22,7 +26,9 @@ class CursorPositionView extends HTMLElement
 
   updatePosition: ->
     if position = @getActiveTextEditor()?.getCursorBufferPosition()
-      @textContent = "#{position.row + 1}:#{position.column + 1}"
+      @row = position.row + 1
+      @column = position.column + 1
+      @textContent = "#{@row}:#{@column}"
     else
       @textContent = ''
```

Comment:
I don't think that's possible because the tile gets `display: none` when it has no / an empty text node, so the tooltip doesn't get triggered.